correct collector ro docker requirement (#898)
splunk/public-o11y-docs: gdi/opentelemetry/install-linux-manual.rst

See :new-page:`docker-compose.yml <https://github.com/signalfx/splunk-otel-collector/blob/main/examples/docker-compose/docker-compose.yml>` in GitHub to download a ``docker-compose`` example.

.. note:: 

>    If you are running the Collector in ``--read-only`` mode and using any Smart Agent receiver's legacy collectd monitor types, you need to provide a writable config directory similar to ``--read-only --tmpfs /usr/lib/splunk-otel-collector/agent-bundle/run:uid=999,gid=999`` (default) or as configured by the Smart Agent extension's ``collectd::configDir`` path.

Reviewer: ```suggestion
   If you are running the Collector in ``--read-only`` mode, and you're using a Smart Agent receiver legacy collectd monitor type, you need to provide a writable config directory. For example, use ``--read-only --tmpfs /usr/lib/splunk-otel-collector/agent-bundle/run:uid=999,gid=999`` (the default) or the directory configured by the Smart Agent extension's ``collectd::configDir`` path.
```
Minor nits. Up to you to fix or ignore. I have approved the PR, but you can update it and resubmit it if you want.
